### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by khanh-devos**
Tasks in this activity:
- Do the styling based on [Zeplin's design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378).
- Add react-circular-progress-bar to build the middle column
- Leverage CSS while styling.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,22 +20,36 @@ function App() {
...
-          <h1>
-            SECRET FOREST
+          <h1 className="header-h1">
+            Bookstore CMS
           </h1>
           <ul className="header-ul">
             <li>
-              <Link className="header-ul-link" to="/">
-                Books
+              <NavLink
```
Comment (iambenkis) [inline src/App.js:30]: - [ ] Kindly make that your design is exactly the same as in the [given design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) 😄  as it is a part of the requirements 👍. Make sure `color` , `padding`.  `margin` , etc , are exactly the same as in the given design 👍  

### Navbar
| Your Design  | Given design |
| :---: | :---: |
| Capture | Capture 1 |

### BookList
| Your Design  | Given design |
| :---: | :---: |
| Capture33 | Capture33 |

### Add book form
| Your Design  | Given design |
| :---: | :---: |
| Capture4  | Capture44
 |

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @khanh-devos 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Gitflow  

## Suggested changes

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.js (hunk 1 above)
Comment (khanh-devos) [reply src/App.js:30]: Hi @iambenkis ,
Thank you for your quick response.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @khanh-devos 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Keep doing the Great work :clinking_glasses: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by khanh-devos** [comment]
Hi @Alfredbis29 
Thank you very much.

**Pull request merged**